downloads/swan-lake-release-notes/swan-lake-2201.9.2/RELEASE_NOTE.md

```diff
@@ -48,6 +48,23 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.2)](
 
 ## Developer tools updates
 
+### Improvements
+
+Introduced the WSDL CLI-tool as an experimental feature which generates Ballerina types from a WSDL file. This tool can be accessed by pulling it from Ballerina Central and running it with an input WSDL file.
```

Comment:
This should be `CLI tool` not `CLI-tool`?

Reply:
will send a PR addressing this.